package.json

```diff
@@ -81,6 +82,7 @@
     "ts-dedent": "^2.2.0",
     "ts-jest": "29.1.4",
     "typescript": "5.4.5",
+    "valibot": "0.31.0-rc.6",
```

Comment:
valibot is planning big breaking changes to its API in v.0.31.0. Since the old API is no longer available, this PR uses the rc version to comply with the new API.

ref: [Migrate from v0.30.0 | Valibot](https://valibot.dev/guides/migrate-from-v0.30.0/)